### PR TITLE
Made App Size 24MB

### PR DESCRIPTION
**Pull request by Gaurav-Kushwaha-1225**
- I have solved the issue https://github.com/VaibhavCodeClub/learn/issues/145
- How I resolved it.
    - Firstly, release app `app-release.apk` wasn't being compiled after running `flutter build apk` or `flutter build apk --release` command in the terminal. It was constantly giving below error.
```
BUILD FAILED in 8s
[!] Your project requires a newer version of the Kotlin Gradle plugin.
    Find the latest version on https://kotlinlang.org/docs/gradle.html#plugin-and-versions, then update project/android/build.gradle:
    ext.kotlin_version = '<latest-version>'
```
- 
    - Then, I saw a solution on StackOverFlow to delete the **android folder in the project directory** and run `flutter create .`
    - After that, `flutter build apk` or `flutter build apk --release` commands worked and gave an .apk file of 35 MB size
    - But, using `flutter build apk --split-per-abi` command also I got an .apk file of 28 MB size. I read about this command on this [StackOverFlow Page](https://stackoverflow.com/questions/49567065/how-to-optimize-the-flutter-app-size) **2nd Answer**.
    - Then, I reduced the size of some images in assets folders like `black.svg`, `onion.png`, etc
    - At last, I got an .apk file of size 24.8 MB. 
- Hope this resolved the Issue #145.

### CODE REVIEW

File: android/app/build.gradle [hunk 1]
```diff
@@ -51,36 +42,26 @@ android {
 
     defaultConfig {
         // TODO: Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
-        applicationId "vdrs.sappu.lafk.learn"
```
Comment (K123Ritesh) [inline android/app/build.gradle:54]: why you changed package name man??

File: android/app/build.gradle [hunk 2]
```diff
@@ -51,36 +42,26 @@ android {
...
+        applicationId "com.example.learn"
         // You can update the following values to match your application needs.
         // For more information, see: https://docs.flutter.dev/deployment/android#reviewing-the-gradle-build-configuration.
         minSdkVersion 21
         targetSdkVersion flutter.targetSdkVersion
         versionCode flutterVersionCode.toInteger()
         versionName flutterVersionName
-        multiDexEnabled true
```
Comment (K123Ritesh) [inline android/app/build.gradle:61]: does multidex enabling really needed??

File: android/app/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android">
```
Comment (K123Ritesh) [inline android/app/src/main/AndroidManifest.xml:1]: Why change in label??

File: android/app/src/main/res/mipmap-hdpi/ic_launcher.png
Comment (K123Ritesh) [inline android/app/src/main/res/mipmap-hdpi/ic_launcher.png:1]: Icon Changed Why??

**Review by K123Ritesh — CHANGES_REQUESTED** [review]

**Comment by K123Ritesh** [comment]
@sapatevaibhav  @Gaurav-Kushwaha-1225 changed icon app label and package name also this is needed or he did it by mistake ??

**Comment by Gaurav-Kushwaha-1225** [comment]
Hi @K123Ritesh,

In above mentioned comment, after running `flutter create .` after deleting the android folder, since the usual command for generating `app-release.apk` i.e. `flutter build apk` wasn't working, a new android folder was created which has default package name, default icon, etc.

But it appears shocking that even during testing the app after the default android folder was generated the app's icon didn't change and was the a\same as before.

Anyways, I will make sure this time and push further commits to resolve this matter.

Thank you.

File: android/app/build.gradle (hunk 2 above)
Comment (sapatevaibhav) [reply android/app/build.gradle]: nope will remove it since app supports minSdk 21

**Review by K123Ritesh — APPROVED** [review]

**Pull request merged**